### PR TITLE
Stars data inside HouseCard and Reviews

### PR DESCRIPTION
**Pull request by AlvesDv**

### CODE REVIEW

File: src/components/HouseCard.jsx [hunk 1]
```diff
@@ -1,6 +1,9 @@
 import { Link } from 'react-router-dom'
+import { FontAwesomeIcon } from '@fortawesome/react-fontawesome'
+import { faStar } from '@fortawesome/free-regular-svg-icons'
```
Comment (ynotrusso) [inline src/components/HouseCard.jsx:3]: wrong package, must use solid

File: src/components/HouseCard.jsx [hunk 2]
```diff
@@ -10,11 +13,21 @@ function HouseCard() {
           className="rounded-t"
         />
         <div className="p-3">
-          <h5 className="font-bold">Phuket, Thailand</h5>
+          <p className="font-bold">Phuket, Thailand</p>
```
Comment (ynotrusso) [inline src/components/HouseCard.jsx:16]: why change everything to paragraphs? You should only add icons in this branch

**Review by ynotrusso — CHANGES_REQUESTED** [review]

File: src/components/HouseCard.jsx (hunk 2 above)
Comment (AlvesDv) [reply src/components/HouseCard.jsx:16]: Should I reverse this changes?

File: src/components/HouseCard.jsx (hunk 1 above)
Comment (AlvesDv) [reply src/components/HouseCard.jsx]: Done  ⭐️

File: src/components/HouseCard.jsx (hunk 2 above)
Comment (ynotrusso) [reply src/components/HouseCard.jsx:16]: Yes :)

**Review by ynotrusso — CHANGES_REQUESTED** [review]

File: src/components/HouseCard.jsx (hunk 2 above)
Comment (AlvesDv) [reply src/components/HouseCard.jsx]: Done  👍

**Review by ynotrusso — APPROVED** [review]

**Review by ynotrusso — APPROVED** [review]

**Pull request merged**